driver/conv_driver.hpp

```diff
@@ -633,6 +816,29 @@ int ConvDriver<Tgpu, Tref>::ParseCmdLineArgs(int argc, char* argv[])
             return 1;
         }
     }
+
+    is_gpualloc = (inflags.GetValueInt("gpualloc") == 1);
+
+    if(is_gpualloc && inflags.GetValueInt("verify") == 1)
+    {
+        std::cerr << "Error: '--gpualloc 1' should not be used with enabled verification. Add "
+                     "'--verify 0' to options."
+                  << std::endl;
```

Comment:
@CAHEK7 IIRC we do not have GPU-based verification yet. Yes, we compute reference on GPU, but buffers are copied back to the host for rms computation. But maybe I am wrong. And yes, this functionality can be added later.

Reply:
Even in case of cpu-based verification - we can generate input data on gpu, generate reference data on gpu and copy only result back to CPU for rms computation. I think it makes sense.
But I completely agree that it should be done in separated PRs.